src/quo/components/wallet/wallet_activity/view.cljs

```diff
@@ -78,7 +78,7 @@
               :width        12
               :height       12
               :borderRadius 6}]}
-    [icon/icon (transaction-icon transaction)
+    [icon/icon (transaction-icon transaction :i/placeholder)
```

Comment:
Some transactions didn't properly set the `transaction-type`, so while rendering this function returned `nil`, causing a crash.

Now the types are being properly set, but I added a placeholder just to make sure the app doesn't crash in case we don't pass the type again.